db/migrate/20220704144504_rename_bactracs_columns_in_spree_shipment.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class RenameBactracsColumnsInSpreeShipment < ActiveRecord::Migration[5.2]
```

Comment:
whether intentional or not, glad you used the same migration version number, so that this doesn't try to re-run

( either way we'll just go with it was intentional 😄 )